src/Pages/About.js

```diff
@@ -44,8 +44,33 @@ const About = () => {
           </button>
         </a>
       </div>
+      <div className="gb-impact">
+        <h2>About Our Impact</h2>
+        <p>
+          Throughout the time Ghost Buses has existed, we've been open to providing people and organizations access to our data to aid others with their causes or purposes. Some towards a more informed and improved Chicago transit system, others towards learning and growth using relatable data. Some of them include:
+        </p>
+        <ul>
+          <li>
+            <a
+              href="https://twitter.com/ctaaction"
+              target="_blank"
+              rel="noreferrer">Commuters Take Action
+            </a> compared scheduled train arrivals with actual run trains.
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/activetrans"
+              target="_blank"
+              rel="noreferrer">Active Transportation Alliance
+            </a> advocated for commuters by revealing ghost buses along specific routes.
+          </li>
+          <li>Ethan A. composed a data analysis paper.</li>
+          <li>Kristen H. analyzed actual and scheduled headways as a learning exercise.</li>
+          <li>Guy who DM'd on Twitter</li>
```

Comment:
I wonder if there's any way we could elaborate on the Twitter interaction in the same way we elaborated in the previous two examples.

Reply:
This has been addressed in [add-impact-to-about-page](https://github.com/chihacknight/ghost-buses-frontend/pull/108). I'll get that merged and then this should be resolved.